dungeon/src/contrib/systems/DecoTestSystem.java

```diff
@@ -0,0 +1,201 @@
+package contrib.systems;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import contrib.components.CollideComponent;
+import contrib.entities.deco.Deco;
+import contrib.entities.deco.DecoFactory;
+import contrib.utils.components.skill.SkillTools;
+import core.Entity;
+import core.Game;
+import core.System;
+import core.components.PositionComponent;
+import core.utils.FontHelper;
+import core.utils.Point;
+import core.utils.Vector2;
+
+public class DecoTestSystem extends System {
+
+  private static final int CHANGE_MODE = Input.Keys.UP;
+
+  private static final int MODE_MODIFY_PLUS = Input.Keys.RIGHT;
+  private static final int MODE_MODIFY_MINUS = Input.Keys.LEFT;
+  private static final int MOVE_DECO = Input.Keys.DOWN;
+  private static final int RAPID_FIRE_THRESHOLD = 5;
+  private static int rapidFireCounter = 0;
+
+  private Entity testEntity;
+  private Deco currentDeco;
+  private Mode currentMode = Mode.ChangeDeco;
+  private final BitmapFont font;
+
+  public DecoTestSystem() {
+    font = FontHelper.getFont("fonts/Roboto-Bold.ttf", 16);
```

Comment:
Ich würde `"fonts/Roboto-Bold.ttf"` in `FontHelper#DEFAULT_FONT` auslagern, so könnte man eine `FontHelper#getDefaultFont(int)` erstellen